### PR TITLE
Feature/refactoringcpf

### PR DESCRIPTION
**Pull request by diggosilva**

### CODE REVIEW

File: CPFCNPJTools/Tests/CPFManagerTests.swift [hunk 1]
```diff
@@ -8,71 +8,71 @@ class CPFValidatorTests: XCTestCase {
     }
     
     func testValidateWhenTheCPFIsNilOrEmpty() {
-        let sut = CPFValidator()
+        let sut = CPFManager()
```
Comment (HelioMesquita) [inline CPFCNPJTools/Tests/CPFManagerTests.swift:11]: extract this `sut` into a variable outside the function to use in all test case

File: CPFCNPJTools/Source/CPFManager.swift [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    /// - Parameter multiplyBy: The starting multiplier value (10 for the first digit, 11 for the second).
+    /// - Returns: The sum of the products of the digits and their respective multipliers.
+    ///
+    /// **Example:**
+    /// ```swift
+    /// let cpfManager = CPFManager()
+    /// let checksum = cpfManager.calculateCPFCheckSum(cpfBaseDigits: [1, 2, 3, 4, 5, 6, 7, 8, 9], multiplyBy: 10)
+    /// print(checksum)  // Result of calculation
+    /// ```
+    private func calculateCPFCheckSum(cpfBaseDigits: [Int], multiplyBy: Int) -> Int {
```
Comment (HelioMesquita) [inline CPFCNPJTools/Source/CPFManager.swift:139]: A private methods doesn't need an example, but that is ok if it has too

Comment (diggosilva) [reply CPFCNPJTools/Source/CPFManager.swift:139]: Done!

File: CPFCNPJTools/Tests/CPFManagerTests.swift (hunk 1 above)
Comment (diggosilva) [reply CPFCNPJTools/Tests/CPFManagerTests.swift:11]: Done!

**Review by HelioMesquita — APPROVED** [review]

**Pull request merged**